src/lib/bridge/transactionMessage/TransactionMessage.ts

```diff
@@ -4,7 +4,7 @@ import type { ITransactionMessageComponentGroup } from './ITransactionMessageCom
 export default class TransactionMessage implements ITransactionMessage {
     constructor(
         public description: string | undefined,
-        public operationGroups: ITransactionMessageComponentGroup[],
+        public transactionGroups: ITransactionMessageComponentGroup[],
```

Comment:
well... 1 transaction message contains only 1 transaction, so it won't have multiple groups of transactions, that's why it was renamed to operationGroups, because there'll be multiple groups of operations. Thoughts?